src/apps/organisations/components/ViewOrganisations.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { APIOrganisation } from "../types";
+import {
+  ContainerSearch,
+} from "../../containers/components";
+import  OrganisationCards  from './organisationCards';
+
+interface Props {
+  organisations: APIOrganisation[];
+  title: string;
+  onSearch: Function;
+  initialQ: string;
+}
+
+const ViewOrganisations: React.FC<Props> = ({
```

Comment:
While it seems to work that typing, e.g., "CIEL" into the search box changes the URL to `http://localhost:8080/user/organisations/?q=CIEL`, if I browse to this URL directly (after logging in) the search field is not filled out.

In other:

1. Login to OCL for OpenMRS
2. Navigate to `http://localhost:8080/user/organisations/?q=CIEL`
3. Note that the search box is blank even though the search *results* are actually filtered

I'm not sure exactly what the cause is here, but we need to take a look into that.

Reply:
Yah, I saw the issue of the search field but I think to fix it I will have to do some changes to the existing Search Container and it is used in all search places, so that might be some work but I will see it I can fix it now or do it as a follow up PR.